### PR TITLE
#163426516 Setup unit testing 

### PR DESCRIPTION
**Pull request by omenkish**
#### What does this PR do?
Setup unit testing

#### How should this be manually tested?
Clone the repo
- run 'npm install' without the quotes
- run 'npm run test' also without the quotes

#### What are the relevant pivotal tracker stories?
[#163426516](https://www.pivotaltracker.com/story/show/163426516)

### CODE REVIEW

**Review by ozone4real — DISMISSED** [review]
Looks good to me

**Review by EfeAgare — DISMISSED** [review]
Looks good to me

File: server/test/funcTest.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { expect } from 'chai';
+
+// dummy function to add numbers
+const addNumbers = (a, b) => a + b;
+
+// test function addNumbers
+describe('A function to add numbers', () => {
+  it('should add two numbera', () => {
```
[the remark this answers is not in the document]
Comment (p-funky) [reply server/test/funcTest.js]: @EfeAgare and @ozone4real you both approved this PR. How did this escape your eyes? 🤔
numbera or `numbers`?

File: server/test/funcTest.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    const result = addNumbers(3, 4);
```
Comment (p-funky) [inline server/test/funcTest.js:9]: What's up with these hound ci issues again please?

**Review by p-funky — APPROVED** [review]

**Pull request merged**